docs/developer_documentation/ref-R.md

```diff
@@ -8,11 +8,13 @@ source("http://molgenis.mydomain.example/molgenis.R")
 ``` 
 at the top of your script and you can connect to a MOLGENIS server. Typically the first thing you do is login and the last thing is logout.
 
-NOTE: For https connections use 
+**NOTE:** For https connections use 
 ```
 eval(expr = parse(text = getURL("https://molgenis.mydomain.example/molgenis.R?molgenis-token=${molgenisToken}")))
 ```  
 
+**NOTE:** We support R-versions earlier or equal to 3.2.x
```

Comment:
Could you make it a little bit clearer that this specifically refers to the R api client?